src/libdrakvuf/private.h

```diff
@@ -276,10 +276,11 @@ struct remapped_gfn
 typedef struct process_data_priv
 {
     char* name;   /* Process name */
-    vmi_pid_t pid ;     /* Process pid */
+    vmi_pid_t pid ;     /* Process pid | tid in linux*/
     vmi_pid_t ppid ;    /* Process parent pid */
     addr_t base_addr ;  /* Process base address */
     int64_t userid ;    /* Process SessionID/UID */
+    vmi_pid_t tid; /* Thread id for Linux*/
```

Comment:
You can add an extra thread-id field for Windows as well, but it should not be of type `vmi_pid_t` because that's just confusing